tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -615,6 +615,36 @@ describe('util/extension-runners/chromium', async () => {
       }),
   );
 
+  it('does pass port to chrome', async () => {
+    const port = 9222;
```

Comment:
This unit test will fail if 9222 is used at any point. You should choose a free port here.

Note that we already have a utility to find a free port, at https://github.com/mozilla/web-ext/blob/1c015188bc215cdbd327bb5d2213727f178d6231/src/firefox/remote.js#L233